openlibrary/plugins/openlibrary/code.py

```diff
@@ -28,6 +28,7 @@
 
 from openlibrary.core.vendors import create_edition_from_amazon_metadata
 from openlibrary.utils.isbn import isbn_13_to_isbn_10, isbn_10_to_isbn_13
+from openlibrary.core.models import Edition
```

Comment:
Ah, it's an issue because of the features override above. I think that refactor should be done separately to see if we can move that lower down (there might be errors). Since this isn't related to this change, let's silence the error on this line for now.
```suggestion
from openlibrary.core.models import Edition  # noqa: E402
```